ci/matrix.xml

```diff
@@ -280,6 +308,13 @@
       <v n="testSuite">testSuite</v>
     </run>
 
+    <!-- Test suite on ECMAScript5 with forced polyfills tasks -->
+    <run task="test-suite-ecma-script5-force-polyfills">
+      <v n="scala">2.12.4</v>
+      <v n="java">1.8</v>
+      <v n="testSuite">testSuite</v>
```

Comment:
This parameter is unneeded.

Reply:
Hum ... I disagree. It's actually used all over in the task's script. In theory we could call that script with `<v n="testSuite">scalaTestSuite</v>`.

Are you saying that because I don't have another `<run>` for that task with a different `testSuite`, which means I could "inline" the value in the script (instead of `$testSuite`)? But if that's the case, we should also apply this reasoning to `scala` and `java`. For `java`, we should even replace it in the entire matrix since we only have `1.8` at the moment.

Reply:
Ok, fair enough.